cogs/sherpmail.py

```diff
@@ -0,0 +1,68 @@
+import discord
+from discord.ext import commands
+from discord import app_commands
+
+# I couldn't work out replying directly to the initial messages so this is not the best
+# Nonetheless it is nearly 6am and I felt the need to complete this
+# I hope it works out and nobody abuses it lol good luck
+
+class sherpMailbox(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.sherp = ""
+    
+    async def cog_load(self):
+        await super().cog_load()
+        print("Sherpmail loaded.")
+
+    @app_commands.command(
+        name = "sherpmail",
+        description = "Leave a message directly to Sherp's DMs"
+    )
+    @app_commands.describe(
+        msg = "The message you would like to leave for Sherp (be respectful)"
+    )
+    async def sendMessage(self, interaction: discord.Interaction, msg: str):
+        # Sherp's Discord ID is currently 212613981465083906
+        self.sherp = await interaction.client.fetch_user(212613981465083906)
+        try:
+            embed = discord.Embed(
+                title = f"📬 You've got mail! **@{interaction.user} | ID: {interaction.user.id}** sent you the following:",
```

Comment:
This seems incorrect, unless discord.py has some special `__repr__` stuff I'm ignorning. I think we should mention interaction.user.global_name and interaction.user.name. The numeric ID isn't too useful IDT.

https://discordpy.readthedocs.io/en/stable/interactions/api.html?highlight=interaction#discord.Interaction.user

Reply:
The unique UID and user would prevent someone from writing stupid things as they would know that their identifiers were also being sent to Sherp. I did rewrite it and have it stated in the footer of the embed instead.